engine/src/main/java/org/terasology/world/block/entity/BlockCommands.java

```diff
@@ -45,22 +44,14 @@
 import org.terasology.utilities.Assets;
 import org.terasology.world.BlockEntityRegistry;
 import org.terasology.world.WorldProvider;
-import org.terasology.world.block.Block;
-import org.terasology.world.block.BlockComponent;
-import org.terasology.world.block.BlockExplorer;
-import org.terasology.world.block.BlockManager;
-import org.terasology.world.block.BlockUri;
+import org.terasology.world.block.*;
```

Comment:
Please don't use start imporst. If you are using IntelliJ, set the threshold value for start imports to something very high to avoid this.